tests/e2e/specs/general-settings-invalid-timezone.test.js

```diff
@@ -0,0 +1,44 @@
+import {
```

Comment:
Three days ago the e2e tests were migrated to playwright 5a838d1bb7e6e1f5aa5f7f8e9924cb8fb67982df

Reply:
Tests converted to Playwright in [2b952d6](https://github.com/WordPress/wordpress-develop/pull/4853/commits/2b952d69762b884cc570c074453c23fbcf6ccc68).